src/main.go

```diff
@@ -9,10 +9,15 @@ import (
 var (
 	version = "dev"
 	commit  = "none"
-	date    = "unknown"
-	builtBy = "unknown"
 )
 
+func truncate_commit(commit string, length int) (short_commit string) {
```

Comment:
Not sure if this is overkill and/or if there's an easier way to do this but if I didn't check then it was failing locally because commit="none" didn't have 12 chars.

Reply:
@eochoa260 - https://go.dev/play/p/p1n5njfblzm
If you are having trouble understanding how this example is working let's have a sync on it.